src/App.js

```diff
@@ -0,0 +1,104 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import { Home } from './components/pages/Home';
+import LogIn from './components/session/LogIn';
+import Logout from './components/session/LogOut';
+import SignUp from './components/session/SignUp';
+import Reservations from './components/pages/Reservations';
+import AddProperties from './components/pages/AddProperties';
+import { Delete } from './components/pages/Delete';
+
+import PersistLogin from './components/session/PersistLogin';
+import PrivateRoute from './routes/PrivateRoute';
+import PublicRoute from './routes/PublicRoute';
+import Sidebar from './components/Sidebar';
+import Details from './components/pages/Details';
+import ReserveForm from './components/pages/ReserveForm';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/skabeo/frontend-capstone/assets/57421823/2a994420-cc2c-4f84-8903-8878a147d094)


That way, your code will meet the latest standards. 💪